### DISCUSSION TITLE
Cannot use my back camera

### DISCUSSION THREAD

**Discussion by NightNice3000** [community/community#137318] [GitHub Education]
When trying to apply for the education program, when choosing the back camera of my phone, the place where the picture should be is only white. And how am i meant to take a good picture with my front camera? I cant see my screen like that.

Edit: Also my school id is designed that all the info is one one side and the expiration date on the other side. So do I upload 2 pictures? If yes how?

**Comment by AkshayMoolya** [comment]
- Check Permissions: Ensure that your browser or app has permission to access your camera. You might need to enable camera access in your phone's settings.
- Close Other Apps: If other apps are using the camera, it might cause issues. Close all other apps that might be using the camera.
- Restart Your Phone: Sometimes, a simple restart can fix the issue.
- Update Browser/App: Make sure your browser or the GitHub app is up to date, as updates often fix bugs.

If the issue persists, try using a different device to complete the application.

**Reply by NightNice3000** [reply]
I used a different device and it still didn't work. Both devices are apple devices. Could that be an issue?

**Reply by Anwishta** [reply]
If the issue persists across different Apple devices, it could be related to the app's compatibility with iOS or a specific bug in the app when used on Apple devices. You can: 

1. Check App Compatibility: Ensure the app is compatible with the iOS version on your devices. Sometimes, apps may not work correctly on certain versions of iOS.

2. Try a Browser: If you're using an app, try applying through the program's website using a mobile browser like Safari.

3. Contact Support: Reach out to the app’s support team or the education program’s help desk to report the issue. They may have a workaround or be aware of compatibility issues with Apple devices.

4. Use a Different OS: If possible, try using an Android device or a computer to complete the application.

**Comment by peter2317238492** [comment]
I encountered the same problem. The rear camera of my iPhone and iPad are not working. How did you solve it?

**Reply by NightNice3000** [reply]
I think I did manage somehow to do a picture with my front camera. Not ideal, but I didn't find another option.

**Comment by MrCoolAndroid** [comment]
I'm having the same problem! I was hoping I could get access to GitHub Education... But there are no solutions to our problems

**Comment by Amank-root** [comment]
I'm also having the same issue for both Android and ios when I try to submit application via website!.
